core/src/main/java/com/yahoo/oak/EntrySet.java

```diff
@@ -152,6 +130,25 @@
         this.keySerializer = keySerializer;
         this.valueSerializer = valueSerializer;
         this.valOffHeapOperator = valOffHeapOperator;
+
+        /*
+         * Key/value reference encoding (64 bit):
```

Comment:
The meaning of number of bits in this comment is not very clear. Is it only upper/lower level limits? In general the comment is not clear.

Reply:
It is the size limits that are derived from the representation size in bits.
> In general the comment is not clear.
Can you elaborate? What sentence does not makes sense to you?

Reply:
It speaks about the limit in the bit size and says it is 28 bit and another 28 bits and then 8 remains. Or something like that. I believe the limit should be between x bits and till y bits depending on abc... Not a specific number. I find the current explanations confusing. I read it and didn't understand what the limits are...

Reply:
I re-edited this comment. I hope it is clearer now.